api/v1/client/endpoint/endpoint_client.go

```diff
@@ -27,7 +27,7 @@ DeleteEndpointID deletes endpoint
 
 Deletes the endpoint specified by the ID. Deletion is imminent and
 atomic, if the deletion request is valid and the endpoint exists,
-deletion will occur even if errors are encounted in the process. If
+deletion will occur even if errors are encountered in the process. If
```

Comment:
Given the code in `api/v1/` is automatically generated with go-swagger, can you fix all the typos in the file `api/v1/openapi.yaml` as well? Otherwise we will undo these fixes when we generate the API side code the next time.